searchwidget/src/main/java/com/harsh/searchwidget/SearchBar.java

```diff
@@ -252,7 +252,7 @@ private void init(AttributeSet attributeSet) {
         navIcon.setOnClickListener(this);
 
         // Placeholder
-        placeholderText = "Search Widget";
+        placeholderText = "@string/app_name";
```

Comment:
Don't give placeholder text the same value as app name. The app name might change in the future. Create a separate variable for it.

Reply:
Okay, got it, new variable added.